newsroom/agenda/utils.py

```diff
@@ -164,6 +164,18 @@ def get_planning_coverages(item, plan_id):
     return [coverage for coverage in item.get("coverages") or [] if coverage.get("planning_id") == plan_id]
 
 
+def get_coverage_status(coverage: Dict[str, Any]) -> str:
+    coverage_status = coverage.get("coverage_status")
+    if coverage_status == "coverage intended":
+        return gettext("Planned")
+    elif coverage_status in ("coverage not decided yet", "coverage upon request"):
```

Comment:
is there any requirement not to have specific status for `on request` coverages?

Reply:
nope, in the templates, there is not any specific requirement for `on request` .

Reply:
would keep those separate then